eventstore/model.go

```diff
@@ -284,6 +303,14 @@ func (t *Txn) FindByID(id core.EntityID, v interface{}) error {
 	if err != nil {
 		return err
 	}
+	if t.model.jsonMode {
+		str := string(bytes)
+		rflStr := reflect.ValueOf(str)
+		reflV := reflect.ValueOf(v)
+		reflV.Elem().Set(rflStr)
+
+		return nil
```

Comment:
Not returning `bytes`?

Reply:
This method returns an `error`. It populates `v` with the searched instance by ID.
Maybe the code confuses a bit since it returns the err of `json.Marshal` for the non-JSONMode case?

Reply:
Oh! Yeah makes sense.